lib/spack/spack/util/web.py

```diff
@@ -226,57 +245,22 @@ def url_exists(url):
         return False
 
 
-def _debug_print_delete_results(result):
-    if 'Deleted' in result:
-        for d in result['Deleted']:
-            tty.debug('Deleted {0}'.format(d['Key']))
-    if 'Errors' in result:
-        for e in result['Errors']:
-            tty.debug('Failed to delete {0} ({1})'.format(
-                e['Key'], e['Message']))
-
-
-def remove_url(url, recursive=False):
```

Comment:
As @scottwittenburg pointed out, that `recursive` argument (and the corresponding logic) is needed.  What is the rationale for removing it?